Amazon/Marketplace/v7153/templates/jfrog-artifactory-core-infrastructure.template.yaml

```diff
@@ -0,0 +1,408 @@
+    Type: AWS::EC2::VPC::Id
+  VpcCidr:
+    Description: CIDR block for the VPC
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
+    Default: 10.0.0.0/16
+    Type: String
+  PrivateSubnet1Cidr:
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
+    Default: 10.0.0.0/19
+    Type: String
+  PrivateSubnet2Cidr:
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
+    Default: 10.0.32.0/19
+    Type: String
+  PrivateSubnet3Cidr:
+    AllowedPattern: ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(1[6-9]|2[0-8]))$
+    ConstraintDescription: CIDR block parameter must be in the form x.x.x.x/16-28
+    Default: 10.0.64.0/19
+    Type: String
+  SubnetIds:
+    Type: List<AWS::EC2::Subnet::Id>
+  DatabaseAllocatedStorage:
+    Type: Number
+  MultiAzDatabase:
+    Description: Choose false to create an Amazon RDS instance in a single Availability Zone.
+    ConstraintDescription: True or False
+    AllowedValues:
+      - "true"
+      - "false"
+    Type: String
+  DatabaseEngine:
+    Type: String
+  DatabaseUser:
+    Type: String
+  DatabasePassword:
+    NoEcho: 'true'
+    Type: String
+  DatabaseInstance:
+    Type: String
+  DatabaseName:
+    Type: String
+  ArtifactoryProduct:
+    Default: JFrog-Artifactory-Pro
+    Type: String
+  ReleaseStage:
+    Default: GA
+    Type: String
+  InstanceType:
+    Default: m5.xlarge
+    Type: String
+  ArtifactoryHostRole:
+    Type: String
+  VolumeSize:
+    Type: Number
+
+Mappings:
+  DatabaseMap:
+    Postgres:
+      Name: postgresql
+      DatabaseVersion: 11.5
+      Driver: "org.postgresql.Driver"
+      Plugin: postgresql-42.2.9.jar
+      PluginURL: https://jdbc.postgresql.org/download/
+      port: "5432"
+      extraDatabaseOps: ""
+  ReleaseStageMap:
+    BETA:
+      ProDockerRepo: "earlyaccess-docker.jfrog.io/jfrog/artifactory-pro"
+      JcrDockerRepo: "earlyaccess-docker.jfrog.io/jfrog/artifactory-jcr"
+      NginxDockerRepo: "earlyaccess-docker.jfrog.io/jfrog/nginx-artifactory-pro"
+    GA:
+      ProDockerRepo: "docker.bintray.io/jfrog/artifactory-pro"
+      JcrDockerRepo: "docker.bintray.io/jfrog/artifactory-jcr"
+      NginxDockerRepo: "docker.bintray.io/jfrog/nginx-artifactory-pro"
+  ProductMap:
+    JFrog-Container-Registry:
+      RepoName: JcrDockerRepo
+    JFrog-Artifactory-Pro:
+      RepoName: ProDockerRepo
+  JavaOptionstoInstance:
+    m5.large:
+      Min: 4
+      Max: 4
```

Comment:
you have to give some description of context - the usage of `min` and `max` heapsize can be valued in MB or GB and you're not making clear that's what it will do.